bot/admin/server/src/test/kotlin/service/ScenarioGroupServiceTest.kt

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright (C) 2017/2022 e-voyageurs technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.tock.bot.admin.service
+
+import ai.tock.bot.admin.scenario.ScenarioGroup
+import ai.tock.bot.admin.scenario.ScenarioGroupDAO
+import ai.tock.shared.exception.scenario.group.ScenarioGroupDuplicatedException
+import ai.tock.shared.exception.scenario.group.ScenarioGroupNotFoundException
+import ai.tock.shared.tockInternalInjector
+import com.github.salomonbrys.kodein.Kodein
+import com.github.salomonbrys.kodein.KodeinInjector
+import com.github.salomonbrys.kodein.bind
+import com.github.salomonbrys.kodein.provider
+import io.mockk.*
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.litote.kmongo.toId
+import java.time.ZonedDateTime
+import kotlin.test.AfterTest
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+class ScenarioGroupServiceImplTest {
```

Comment:
```suggestion
class ScenarioGroupServiceTest {
```
CAr il n'y a pas d'interface de toute manière sur cette classe